### PR TITLE
Clarify symptoms and repeats

### LINKED ISSUE #8: Non-duplication of individuals
The OR estimator is unbiased when you allow every person an independent (or multiple independent) opportunities to enter the study as infected or not.

In the limit of a large population, this isn't a problem. In smaller populations, what happens?

### PR DESCRIPTION
**Pull request by swo**
Note that I'm simplifying the derivation by cutting out the layer of symptoms. I intend to show in another PR that symptoms are actually a confusing concept.

### CODE REVIEW

**Comment by swo** [comment]
@eschrom This is not legible as of https://github.com/CDCgov/cfa-tnd-ve-widget/pull/24/commits/7059b7a8c03ee2a0ef558a7610fdb550fc1b5467. I strongly recommend rendering math with mkdocs, not GitHub.

File: docs/primer_short.md [hunk 1]
```diff
@@ -2,109 +2,257 @@
...
+
+<!-- TO DO: Better name? -->
+
+In this model:
+
+$$
+\mathrm{ORE} = \mathrm{RR} \cdot \frac{\lambda^{T-}_U}{\lambda^{T-}_V}
+$$
+
+Thus, so long as the vaccinated and unvaccinated are tested for test-negative conditions at the same rate, then the TND odds ratio is an unbiased estimator of the relevant VE risk ratio (subject also the requirement that rates be nonzero and in the limit of large populations).
```
Comment (eschrom) [inline docs/primer_short.md]: ... subject also _to_ the requirement that ...

File: docs/primer_short.md [hunk 2]
```diff
@@ -2,109 +2,257 @@
...
 
-- The population of interest are those individuals who received a diagnostic test, usually becomes they develop some symptoms, seek a test, and qualify for that test.
-- The main covariate of interest is vaccination status.
-- The outcome of interest is the test result (positive or negative).
+In the simplest derivation, we assume that:
 
-The data available in a TND ultimately reduces to a 2x2 table of counts:
+- Test performance is perfect.
+- There is some population of $n_V$ vaccinated individuals and $n_U$ unvaccinated individuals that are _a priori_ eligible for inclusion in the TND.
+- Testing is a Poisson process. Each vaccinated individual receives tests for the test-positive condition at rate $\lambda^{T+}_V$ and for the test-negative condition at rate $\lambda^{T-}_V$.
```
Comment (eschrom) [inline docs/primer_short.md]: A Poisson testing process seems unrealistic. Typically, whether/when an individual gets tested depends on whether/when they get sick and seek care. This period of illness (if it occurs at all) means that the rate at which an individual receives tests for either condition is highly variable, not constant, in time.

In the past, we've said that an individual's expected number of tests is something like:
(Exposure Rate) * P(infection | exposure) * P("symptoms" | infection) * P(test | "symptoms") * Study Duration

Does the Poisson assumption offer any unique benefits to your derivation?
- If no, then perhaps you want to redefine $\lambda$ as the above product, rather than a Poisson rate.
- If yes, we better carefully understand why, since the Poisson assumption is probably far from true.

File: docs/primer_short.md (hunk 1 above)
Comment (eschrom) [inline docs/primer_short.md]: > so long as the vaccinated and unvaccinated are tested for test-negative conditions at the same rate, then the TND odds ratio is an unbiased estimator of the relevant VE risk ratio

I'm afraid this sentence runs counter to the whole point of TND. Quite likely, the vaccinated and the unvaccinated do _not_ get tested at the same rate. Because an individual's testing rate involves the psychological choice to seek a test/care, and vaccination or lack thereof indicates differing psychologies, we expect that $\lambda_U^{T-} \neq \lambda_V^{T-}$, such that $\frac{\lambda_U^{T-}}{\lambda_V^{T-}} \neq 1$

However, far from creating bias in the ORE, this inequality _rescues_ the ORE from bias. This is because the first term of the ORE, $RR = \frac{\lambda_V^{T+}}{\lambda_U^{T+}}$, suffers from the same bias due to the differing psychologies of the vaccinated vs. the unvaccinated. The second term of the ORE undoes this bias.

In other words, the directly calculated RR, $\frac{\lambda_V^{T+}}{\lambda_U^{T+}}$, differs from 1 because of the biological effect of the vaccine _and_ the psychology of vaccination/care-seeking/testing. The inclusion of psychology is the bias that makes RR alone an unsuitable estimate of VE. Thus, we multiply RR by $\frac{\lambda_U^{T-}}{\lambda_V^{T-}}$, which differs from 1 _only_ because of the psychology of vaccination/care-seeking/testing. This multiplication creates the ORE and undoes the bias due to psychology, leaving only the biological effect of the vaccine.

Thus, the TND odds ratio (instead of the risk ratio) is an unbiased estimator of the relevant VE risk ratio _precisely because_ the vaccinated and unvaccinated are tested at different rates.

File: docs/primer_short.md [hunk 3]
```diff
@@ -2,109 +2,257 @@
...
+
+(To check this result, treat exposure and testing as one-time possibilities, replacing $\lambda_V^E \to P[E|V]$ and $\lambda_V^{T+} \to P[T,I|V]$.)
+
+In this case, the ORE is:
+
+$$
+\frac{P[T,I|E,V]}{P[T,I|E,U]} \cdot \frac{\lambda^E_V}{\lambda^E_U} \cdot \frac{\lambda^{T-}_U}{\lambda^{T-}_V}
+$$
+
+Thus, if the vaccinated and unvaccinated have equal exposure rates, then the TND odds ratio will be an unbiased estimate of protection from vaccine _conditioned on exposure_ (subject again to the other assumptions above). If the vaccinated are exposed at a higher rate, then TND will underestimate VE.
```
Comment (eschrom) [inline docs/primer_short.md]: If the vaccinated and the unvaccinated have equal exposure rates, i.e. $\frac{\lambda^E_V}{\lambda^E_U} = 1$, then why do you need to say, "_conditioned on exposure_?" Exposure cancels out and disappears, so there is not any conditioning on exposure, right?

Comment (eschrom) [inline docs/primer_short.md]: > If the vaccinated are exposed at a higher rate, then TND will underestimate VE.

This is not necessarily true. Suppose the vaccinated are exposed to the test-positive condition at a higher rate, but they are also exposed to the test-negative condition(s) at a higher rate. Then these effects will cancel out inside the ORE, leaving the ORE as an unbiased estimator of VE again. This is addressed in [the longer primer](https://github.com/CDCgov/cfa-tnd-ve-widget/blob/main/docs/primer.md).

File: docs/primer_short.md [hunk 4]
```diff
@@ -2,109 +2,257 @@
...
+
+## Conditioning on symptoms
+
+**Proposition: TND does not account for testing behavior.**
+
+It is commonly held that TNDs mitigate biases due to differences in testing behavior between the vaccinated and unvaccinated. This cannot be generally true: if symptoms information is not part of the analysis, then for the purposes of analysis, testing behavior and disease progression are perfectly confounded. We cannot distinguish how the vaccine reduces symptoms versus how the vaccinated and unvaccinated seek rates at different rates for different symptoms.
```
Comment (eschrom) [inline docs/primer_short.md]: > We cannot distinguish how the vaccine reduces symptoms versus how the vaccinated and unvaccinated seek **tests** at different rates for different symptoms. 

The TND is designed to do (nearly) exactly this. The inclusion of the test-negative condition (i.e. the second term of the ORE) removes the confounding influence of "how the vaccinated and the unvaccinated seek tests at different rates" from the risk ratio, leaving only "how the vaccine reduces symptoms." So I think this sentence is too strong a statement.

Where things might get interesting, though, is where you consider multiple different symptoms. TND typically assumes that "symptoms" is one thing: acute respiratory illness - either you have it, or you don't. But you are about to dismantle this overly simplified assumption. So as I read, I will think carefully about how expanding our universe from one binary symptom to many continuous symptoms might affect TND.

File: docs/primer_short.md [hunk 5]
```diff
@@ -2,109 +2,257 @@
...
+\lambda^{T+}_V &= P[T,I|E,V] \cdot \lambda^E_V \\
+&= \sum_i P[T,S_i,I|E,V] \cdot \lambda^E_V \\
+&= \sum_i P[T|S_i,I,E,V] P[S_i,I|E,V] \cdot \lambda^E_V
+\end{aligned}
+$$
+
+Note that $E \cap I = I$, so that $P[T|S_i,I,E,V] = P[T|S_i,I,V]$. Assume that the probability of testing depends only on symptoms and vaccination (i.e., testing and infection status are conditionally independent given symptoms) so that $P[T|S_i,I,V] = P[T|S_i,V]$. Similarly, $P[S_i,I|E,V] = P[S_i|I,V] P[I|E,V]$. Thus:
+
+$$
+\lambda^{T+}_V = \sum_i P[T|S_i,V] \cdot P[S_i|I,V] \cdot P[I|E,V] \cdot \lambda^E_V
```
Comment (eschrom) [inline docs/primer_short.md]: I like this - this is exactly the sort of thing I suggested replacing the Poisson assumption with from the very beginning.

However, I think something isn't right about the summation. Suppose I am very test-happy: if I get a fever over 102, I will seek a test with probability 1, and if I cough more than once every ten minutes, I will seek a test with probability 1. Now suppose I have a fever over 102 _and_ I am coughing twice every ten minutes. In practice, this will have the same result as either symptom on its own: I will go to the doctor and seek a test. But the summation says that with both symptoms, I will seek a test at twice the rate as if I only had one symptom or the other. What would "twice the rate" mean in practice?

File: docs/primer_short.md [hunk 6]
```diff
@@ -2,109 +2,257 @@
...
+$$
+\frac{\sum_i P[T|S_i,V] P[S_i|I,V]}{\sum_i P[T|S_i,U] P[S_i|I,U]}
+\cdot \frac{P[I|E,V]}{P[I|E,U]}
+\cdot \frac{\lambda^E_V}{\lambda^E_U}
+\cdot \frac{\sum_i P[T|S_i,U] \lambda^{S_i-}_U}{\sum_i P[T|S_i,V] \lambda^{S_i-}_V}
+$$
+
+The first fraction is relative risk of a seeking and receiving a positive test, given infection, in the vaccinated relative to the unvaccinated. The second fraction is the relative risk of infection given exposure. The third fraction is the relative rate of exposure. The fourth fraction is the relative risk of seeking and receiving a negative test.
+
+This ORE does not reduce, even under some simplifying assumptions. For example, even if:
```
Comment (eschrom) [inline docs/primer_short.md]: This ORE will not reduce because the products inside the sums in the first term are different from the products inside the sums in the fourth term. If $P[T|S_i,V]$ and $P[T|S_i,U]$ were not inside sums, they would cancel out between the first and the fourth term. The sums arise because we are considering multiple symptoms. This is the crux of how considering multiple symptoms _might_ sabotage TND.

I say "_might_" because, as noted above, I'm not convinced that sums are the right way to handle multiple symptoms. If sums are not appropriate, then some other handling of multiple symptoms might allow the ORE to reduce quite naturally.

File: docs/primer_short.md [hunk 7]
```diff
@@ -2,109 +2,257 @@
...
-    - $P[N|T_I,V] = 1 - p_\mathrm{sens}$
-    - $P[N|T_X,V] = p_\mathrm{spec}$
-  - We _a priori_ assume that test performance is unrelated to vaccination status, e.g., $P[P|T_I,V]=P[P|T_I,U]$.
+### Counterexample
 
-#### Quantity of interest
+To see why TND cannot generally correct for test-seeking behavior, consider a simple case where there are 3 types of symptoms:
 
-We are interested in protection against symptomatic disease, conditioned on exposure:
+- 0: Neither the vaccinated not unvaccinated seek a test, so that $P[T|S_0,V]=P[T|S_0,U]=0$
```
Comment (eschrom) [inline docs/primer_short.md]: Neither the vaccinated _nor_ unvaccinated

File: docs/primer_short.md [hunk 8]
```diff
@@ -2,109 +2,257 @@
...
+- 1: Everyone seeks a test, with equal probability $P[T|S_1]$.
+- 2: The vaccinated seek a test; the unvaccinated do not.
 
-```math
-\mathrm{VE}_{SP} = 1 - \frac{P[S_I|E,V]}{P[S_I|E,U]}
-```
+$$
+\begin{aligned}
+&\quad \frac{P[T|S_1] P[S_1|I] + P[T|S_2,V]P[S_2|I,V]}{P[T|S_1] P[S_1|I,U]}
```
Comment (eschrom) [inline docs/primer_short.md]: More evidence that summation isn't the right approach to multiple symptoms: the numerator of the first term represents the probability of seeking a test given infection and vaccination, but this term can exceed 1, so it can't really represent a probability.

I think we better come to a conclusion on the summation approach before going further, because the summation may be a crucial part of the conclusion that the ORE will no longer reduce with multiple symptoms.

**Review by eschrom — CHANGES_REQUESTED** [review]
Uh oh! I found points of disagreement in many places - enough that we should probably work through them before trying to push further. Let me know if you want to talk in person!

File: docs/primer_short.md (hunk 3 above)
Comment (swo) [reply docs/primer_short.md]: Better language is "bias downward," because different biases can point in different directions, leading to overall under/overestimate.

File: docs/primer_short.md (hunk 4 above)
Comment (swo) [reply docs/primer_short.md]: Agree that this sentence is confusing/misleading/false(?) in the context of the monolithic/binary symptoms definition. I'll change the order of concepts

File: docs/primer_short.md (hunk 5 above)
Comment (swo) [reply docs/primer_short.md]: The symptom statuses are disjoint. So the status are like "cough and no fever," "fever and no cough," "cough and fever," "neither cough nor fever"

Thus $\bigcup_i S_i = \Omega$, and $S_i \cap S_j = \varnothing$ for $i \neq j$, and for any event $X$, we have $X = \bigcup_i (X \cap S_i)$. This is why I can say $P[T|V] = P[T,V]P[V] = \sum_i P[T,S_i,V] P[V]$, and similar.

The word disjoint is in there somewhere but this tells me I need to remind/re-emphasize.

The derivation doesn't require --and I don't want to make-- any assumptions about how symptoms synergize (or not).

File: docs/primer_short.md (hunk 2 above)
Comment (swo) [reply docs/primer_short.md]: My point with a Poisson process is how you get to replacing a random number of tests per individual (I might test 0, 1, 2, ... times) with a scalar rate like $\lambda$.

For the avoidance of doubt, I use "Poisson process" to mean that the number of tests each individual gets in the study period is Poisson distributed (with mean $\lambda^{T+}$) and the times of those tests are uniformly distributed through the study period.

I suppose I could be more careful and say that it could really be an inhomogeneous Poisson process, and the unconditional risk of receiving a positive test can vary through time.

I definitely get to that chain of exposure * infection|exposure ... etc. in later sections. But I wanted to start by keeping it simple and say "un/vaccinated people receive positive/negative tests at some rate."

Does this make sense? I want to make sure there isn't something else I'm missing here.

File: docs/primer_short.md (hunk 1 above)
Comment (swo) [reply docs/primer_short.md]: After the symptoms stuff below, I'm now so skeptical about this claim. Let's talk more after we sort that stuff out.

To my mind, the primary utility of the negative tests is to transform from *numbers* of tests to *per capita rates* of tests. In other words, I mostly want them because they have the $P[V]$ and $P[U]$ terms to cancel out the ones from the positive tests. The primarily help you see if you're getting more unvaccinated positives because the vaccine works or just because there are more unvaccinated people. In this sense, deviations from $\lambda^{T-}_U = \lambda^{T-}_V$ are bad.

I agree with you it's appealing that we could use the fact that $\lambda^{T-}_U \neq \lambda^{T-}_V$ to "undo" the different test-seeking behavior, but I'm just so skeptical about that now.

File: docs/primer_short.md (hunk 2 above)
Comment (eschrom) [reply docs/primer_short.md]: I think an inhomogeneous Poisson process would work. I just think the assumptions of a standard Poisson invoke some unnecessary question/doubt from the start. 

Just like there is a devil lurking in the details of the simple assumption about symptoms, I'll bet there is also a devil lurking in the details of timing/rates. It warrants some care, even though it's not your focus here.

File: docs/primer_short.md (hunk 1 above)
Comment (eschrom) [reply docs/primer_short.md]: Your skepticism is legitimate! I just wouldn't reveal that skepticism until after the multiple-symptoms framework is introduced, since that is the basis for your skepticism.

File: docs/primer_short.md (hunk 3 above)
Comment (eschrom) [reply docs/primer_short.md]: A lesser-known but required assumption of the TND is that any differences in exposure to the test-positive condition between the vaccinated and the unvaccinated is mirrored for the test-negative condition(s). This assumption prevents the bias you are concerned about.

That said, this assumption is perhaps unlikely to be true.

File: docs/primer_short.md (hunk 4 above)
Comment (eschrom) [reply docs/primer_short.md]: Yes, good call - the order of concepts might be primary thing to change in all of this

File: docs/primer_short.md (hunk 5 above)
Comment (eschrom) [reply docs/primer_short.md]: Excellent - I see the emphasis on "disjoint" now. I will puzzle through the multi-symptom math again from the beginning.

File: docs/primer_short.md (hunk 2 above)
Comment (swo) [reply docs/primer_short.md]: Inhomogeneous Poisson made this cleaner: this just means that each individual person's number of exposures and test-negative conditions and (therefore by some extensions) test-positive and -negative tests *over the course of the study* are each Poisson distributed. This is a lot clearer than saying "there is this constant *rate* over the whole study/

**Comment by swo** [comment]
@eschrom I'm breaking this down into a mutated `primer_short.md` (which I'll likely adapt in yet another PR, to make it a useful piece of explanation/documentation for the widget) and a separate, more compact `symptoms.md` (which will be a pointed refutation of the symptoms idea).

**Comment by eschrom** [comment]
Sounds good to me!

File: docs/primer_short.md (hunk 3 above)
Comment (swo) [reply docs/primer_short.md]: I'll try to clarify:

Thus, if the vaccinated and unvaccinated have equal exposure rates, then the TND odds ratio will be an unbiased estimate of protection from vaccine _conditioned on exposure_, that is, that we can estimate the conditional risk ratio $P[T,I|E,V]/P[T,I|E,U]$ rather than the unconditional ratio $P[T,I|V]/P[T,I|U]$.

I see how this can sound kind of tautological, but people are often tempted to think of VE as reduction in risk given that you were exposed, rather than a reduction in risk on whatever background of risk existing in the VE study.
